cpp/include/cudf/detail/scatter.cuh

```diff
@@ -118,6 +119,20 @@ struct column_scatterer_impl<string_view, MapIterator> {
   }
 };
 
+template <typename MapIterator>
+struct column_scatterer_impl<list_view, MapIterator> {
+  std::unique_ptr<column> operator()(column_view const& source,
+                                     MapIterator scatter_map_begin,
+                                     MapIterator scatter_map_end,
+                                     column_view const& target,
+                                     rmm::mr::device_memory_resource* mr,
+                                     cudaStream_t stream) const
```

Comment:
Thank you for the advice and the heads-up, @harrism. 

Since `cudf/detail/scatter.cuh` will be updated to use `rmm::cuda_stream_view` in #6648, I'm not completely clear on how to proceed with this recommendation. I can make the corresponding change in `cudf/lists/detail/scatter.cuh` simply enough, but what about `cudf/detail/scatter.cuh`?

Reply:
I don't think you need to change any existing APIs, only the APIs you are adding. Existing APIs will be updated in my PRs. If My PRs merge first, you will just need to resolve conflicts. If yours merges first, I will resolve conflicts.

Reply:
Kewl. I've made this change for all functions introduced in `cudf/lists/detail/scatter.cuh`.